.github/workflows/auto-merge-dependabot.yml

```diff
@@ -0,0 +1,66 @@
+                  steps.metadata.outputs.dependency-names == '@babel/plugin-syntax-jsx' ||
+                  steps.metadata.outputs.dependency-names == '@babel/polyfill' ||
+                  steps.metadata.outputs.dependency-names == '@types/classnames' ||
+                  steps.metadata.outputs.dependency-names == '@types/dinero.js' ||
+                  steps.metadata.outputs.dependency-names == '@types/dompurify' ||
+                  steps.metadata.outputs.dependency-names == '@types/gtag.js' ||
+                  steps.metadata.outputs.dependency-names == '@types/jest' ||
+                  steps.metadata.outputs.dependency-names == '@types/jest-environment-puppeteer' ||
+                  steps.metadata.outputs.dependency-names == '@types/jquery' ||
+                  steps.metadata.outputs.dependency-names == '@types/lodash' ||
+                  steps.metadata.outputs.dependency-names == '@types/puppeteer' ||
+                  steps.metadata.outputs.dependency-names == '@types/react' ||
+                  steps.metadata.outputs.dependency-names == '@types/react-dom' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__block-editor' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__blocks' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__data' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__data-controls' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__editor' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__notices' ||
+                  steps.metadata.outputs.dependency-names == '@typescript-eslint/eslint-plugin' ||
+                  steps.metadata.outputs.dependency-names == '@typescript-eslint/parser' ||
+                  steps.metadata.outputs.dependency-names == 'chalk' ||
+                  steps.metadata.outputs.dependency-names == 'circular-dependency-plugin' ||
+                  steps.metadata.outputs.dependency-names == 'commander' ||
+                  steps.metadata.outputs.dependency-names == 'copy-webpack-plugin' ||
+                  steps.metadata.outputs.dependency-names == 'eslint-import-resolver-typescript' ||
+                  steps.metadata.outputs.dependency-names == 'gh-pages' ||
+                  steps.metadata.outputs.dependency-names == 'markdown-it' ||
+                  steps.metadata.outputs.dependency-names == 'promptly' ||
+                  steps.metadata.outputs.dependency-names == 'react-docgen' ||
+                  steps.metadata.outputs.dependency-names == 'wp-types'
+              }}"
+
+              run: |
+                gh pr edit --add-label 'dependencies-auto-merged' "$PR_URL"
+                gh pr merge --auto --squash "$PR_URL"
+                gh pr review --approve "$PR_URL"
```

Comment:
I'm a bit confused by these two lines. The way I understand is that we are merging the PR first (`gh pr merge`) and then approving it (`gh pr review --approve`)? Wouldn't it be better the other way around?

Reply:
Hey! `gh pr merge --auto --squash "$PR_URL"` is not merging the PR, it is turning on auto-merge with the `--auto` option: in this case, the PR will be merged [only after the necessary requirements are met](https://cli.github.com/manual/gh_pr_merge) (all checks passed): but I totally agree this can be confusing at a first glance, so I'm going ahead and inverting the lines for the sake of clarity :)